cocotb/_build_libs.py

```diff
@@ -261,76 +280,58 @@ def get_ext():
     if sys.platform == "darwin":
         cfg_vars["LDSHARED"] = cfg_vars["LDSHARED"].replace("-bundle", "-dynamiclib")
 
-    share_dir = os.path.relpath(os.path.join(os.path.dirname(__file__), "share"))
+    share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "share"))
     share_lib_dir = os.path.relpath(os.path.join(share_dir, "lib"))
+    share_def_dir = os.path.abspath(os.path.join(share_dir, "def"))
     include_dir = os.path.relpath(os.path.join(share_dir, "include"))
 
+    _gen_import_libs(share_def_dir)
+
     ext = []
 
     logger.info("Compiling interface libraries for cocotb ...")
 
     #
     #  Icarus Verilog
     #
-    icarus_compile = True
     icarus_extra_lib = []
     icarus_extra_lib_path = []
     logger.info("Compiling libraries for Icarus Verilog")
     if os.name == "nt":
-        iverilog_path = find_executable("iverilog")
-        if iverilog_path is None:
-            logger.warning(
-                "Icarus Verilog executable not found. No VPI interface will be available."
-            )
-            icarus_compile = False
-        else:
-            icarus_path = os.path.dirname(os.path.dirname(iverilog_path))
-            icarus_extra_lib = ["vpi"]
-            icarus_extra_lib_path = [os.path.join(icarus_path, "lib")]
-
-    if icarus_compile:
-        ext += _get_common_lib_ext(include_dir, share_lib_dir, sim_define="ICARUS")
-        icarus_vpi_ext = _get_vpi_lib_ext(
-            include_dir=include_dir,
-            share_lib_dir=share_lib_dir,
-            sim_define="ICARUS",
-            extra_lib=icarus_extra_lib,
-            extra_lib_dir=icarus_extra_lib_path,
-        )
-
-        ext.append(icarus_vpi_ext)
+        icarus_extra_lib = ["icarus"]
```

Comment:
I'm confused, aren't you still wanting to load `libvpi` and not `libicarus` at runtime?

Reply:
No. Never needed on posix.

Reply:
The dll name is in the def file.

Reply:
Ah, that's what I missed - thanks!